src/Shared/Recruit.Vacancies.Client/Domain/Entities/VacancyIdentifier.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+
```

Comment:
Can we remove the unused using directive?

Reply:
👍